sim/warlock/warlock.go

```diff
@@ -77,6 +80,7 @@ func (warlock *Warlock) Initialize() {
 
 	// base scaling value for a level 85 warlock
 	warlock.ScalingBaseDamage = 962.335630
+	warlock.SoulShards = 4
```

Comment:
This should be done in the Reset method to make sure it sets it to 4 at start of next iterations as well